assets/boilerplate/theme.json

```diff
@@ -0,0 +1,303 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/theme.json",
+    "settings": {
+        "appearanceTools": true,
+        "border": {
+            "color": true,
+            "radius": true,
+            "style": true,
+            "width": true
+        },
+        "color": {
+            "custom": true,
+            "customGradient": true,
+            "link": true,
+            "palette": [
+                {
+                    "color": "#776c4e",
+                    "name": "Primary",
+                    "slug": "primary"
+                },
+                {
+                    "color": "#a19982",
+                    "name": "Secondary",
+                    "slug": "secondary"
+                },
+                {
+                    "color": "#000000",
+                    "name": "Foreground",
+                    "slug": "foreground"
+                },
+                {
+                    "color": "#ffffff",
+                    "name": "Background",
+                    "slug": "background"
+                },
+                {
+                    "color": "#f1ede6",
+                    "name": "Tertiary",
+                    "slug": "tertiary"
+                }
+            ]
+        },
+        "custom": { 
+            "gap": {
+                "horizontal": "min(30px, 5vw)",
+                "vertical": "min(30px, 5vw)"
+            }
+        },
+        "layout": {
+            "contentSize": "620px",
+            "wideSize": "1000px"
+        },
+        "spacing": {
+            "blockGap": true,
+            "margin": true,
+            "padding": true,
+            "units": [
+                "%",
+                "px",
+                "em",
+                "rem",
+                "vh",
+                "vw"
+            ]
+        },
+        "typography": {
+            "customFontSize": true,
```

Comment:
customFontSize is enabled by default, this can be removed.
https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/#typography